### PR TITLE
Updating the name of Azure Active Directory to Microsoft Entra ID

### PR DESCRIPTION
**Pull request by lidiazuin**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates.

You can view the updated docs at https://neo4j-docs-operations-1187.surge.sh

File: modules/ROOT/pages/tutorial/tutorial-sso-configuration.adoc [hunk 1]
```diff
@@ -234,33 +234,33 @@ For details, see https://learn.microsoft.com/en-us/azure/active-directory/develo
...
+Entra App Roles allow a layer of separation between Neo4j roles and AD Groups.
 When App Roles are used, only the roles relevant to Neo4j are sent in the JWT token.
 This prevents leaking permissions between applications.
 JWT tokens also have a limitation of 200 roles per token per user, which can be avoided by sending only the relevant App Roles.
 
-Details about Azure App Roles can be found in the https://learn.microsoft.com/en-us/azure/active-directory/develop/howto-add-app-roles-in-azure-ad-apps[Microsoft documentation].
+Details about Entra ID App Roles can be found in the https://learn.microsoft.com/en-us/entra/identity-platform/howto-add-app-roles-in-apps[Microsoft documentation].
 
-==== Using Azure AD Groups directly
+==== Using Entra AD Groups directly
```
Comment (renetapopova) [inline modules/ROOT/pages/tutorial/tutorial-sso-configuration.adoc:250]: I am not sure if they still call them AD groups.

Comment (lidiazuin) [reply modules/ROOT/pages/tutorial/tutorial-sso-configuration.adoc:250]: You're right! It should be just Entra. I will fix that. Thanks!

**Review by renetapopova — APPROVED** [review]
It looks good to me, but I think Cypher Operations should approve it as well. I am also not sure if we need to update the screenshots.

**Comment by lidiazuin** [comment]
> It looks good to me, but I think Cypher Operations should approve it as well. I am also not sure if we need to update the screenshots.

Ok! I will ask someone from cypher operations to review. About the screenshots, I checked them and the interface didn't change.

**Pull request merged**